scripts/dev/build_release.py

```diff
@@ -100,6 +100,72 @@ def call_tox(
         env=env, check=True)
 
 
+def fixup_qtwebengine_sandboxing():
+    """WORKAROUND for PyInstaller problems with sanboxing on macOS.
+
+    https://github.com/pyinstaller/pyinstaller/pull/6903
+    https://github.com/qutebrowser/qutebrowser/issues/7278
+    """
+    qute_bundle = pathlib.Path('dist') / 'qutebrowser'
+    app_bundle = pathlib.Path('dist') / 'qutebrowser.app'
+
+    print(f"Attempting to fix up QtWebEngine in .app bundle: {app_bundle}")
+
+    app_res = app_bundle / "Contents" / "Resources"
+    qt_resdir = app_res / "PyQt6" / "Qt6" / 'lib' / 'QtWebEngineCore.framework' / 'Resources'
+
+    qute_lib_dir = qute_bundle / "PyQt6" / "Qt6" / 'lib' / 'QtWebEngineCore.framework' / 'Resources'
+
+    for resource in qt_resdir.iterdir():
+        if resource.suffix != ".pak":
+            continue
+
+        name = resource.name
+        resource.unlink()
+        shutil.copy(qute_lib_dir / name, resource)
+
+    top_level = app_bundle / "Contents" / "MacOS"
+    qt_libdir = top_level / "PyQt6" / "Qt6" / "lib"
+
+    # For collected Qt .framework bundles, replace the copy of lib in top-level directory
+    # with symlink to the copy from the .framework bundle
+    for framework in qt_libdir.iterdir():
+        if not framework.is_dir():
+            continue
+        if not framework.match("**/*.framework"):
+            continue
```

Comment:
```suggestion
    for framework in qt_libdir.rglob("*.framework"):
```

should do the same I believe? This wouldn't skip dirs, but I suppose we shouldn't have `*.framework` files which are non-dirs anyways?